src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+
+import "./NavSearchResult.scss";
+
+interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
```

Comment:
Is `e` actually optional, or is it always provided?

```suggestion
  onClick?: (e: React.MouseEvent) => void;
```

Reply:
when the event is clicked, nothing happens. Should there be some kind of logic?

Reply:
Good question. I would be searching the codebase for usages of this prop+this component and see if/what they are trying to do with it. It might just be a legacy hangover that can be removed potentially.

If it's still being used, and those usages seem legitimate, then we should probably be running the handler.

Reply:
Looking at the usage, it's passed in to the `onClick` of the main `div`, so that's how it's being 'called' here. And you can see where it is being passed from `src/components/molecules/NavSearchBar/NavSearchBar.tsx`, so it's definitely used.

---

My question above was more specifically about the type that the `div`'s `onClick` event passes for that parameter. But thinking about it more now.. by having it optional like this it means we don't need to define it in our own handlers if we never need to use it, which is correct. So should be fine to leave as is.